project-data/schemas/project.ts

```diff
@@ -0,0 +1,238 @@
+      description: "Project description",
+      group: "info",
+      type: "string",
+      validation: (r) => r.required().min(20),
+    },
+    {
+      name: "registry",
+      description: "Verra, Gold Standard, or EcoRegistry",
+      group: "info",
+      placeholder: "VCS",
+      type: "string",
+      options: {
+        list: [
+          { title: "Verra", value: "VCS" },
+          { title: "Gold Standard", value: "GS" },
+          { title: "EcoRegistry", value: "ECO" },
+        ],
+      },
+      validation: (r) => r.required(),
+    },
+    {
+      name: "registryProjectId",
+      description:
+        "Official identifier as it appears in the registry. Do not include a prefix.",
+      placeholder: "432",
+      group: "info",
+      type: "string",
+      validation: (r) =>
+        r.min(1).regex(/^[0-9]*$/, { name: "Numeric characters only" }),
+    },
+    defineField({
+      name: "id",
+      description:
+        "Project identifier, unique, derived from [registry]-[registryProjectId]",
+      group: "info",
+      type: "slug",
+      options: {
+        source: (doc) => `${doc.registry}-${doc.registryProjectId}`,
+        slugify: (str) => str.toUpperCase(),
+      },
+      validation: (r) =>
+        r.custom((val, ctx) => {
+          const str = val?.current;
+          const registryProjectId = ctx.document?.registryProjectId as
+            | string
+            | undefined;
+          const registry = ctx.document?.registry as string | undefined;
+          if (!str) return "Please click Generate";
+          if (!registry) return "Needs registry prefix";
+          if (!registryProjectId) return "Needs Registry Project Id";
+          if (str !== `${registry}-${registryProjectId}`)
+            return "Please re-generate";
+          return true;
+        }),
+    }),
+    defineField({
+      type: "reference",
+      name: "methodology",
+      to: [{ type: "methodology" }],
+      description: "Project methodology",
+      group: "info",
+    }),
+    {
+      name: "country",
+      description: "ISO-3166 English Short Name of the country",
+      type: "string",
+      group: "location",
+      validation: (r) => r.required(),
+      options: {
+        list: countries.map((c) => ({ title: c.name, value: c.name })),
+      },
+    },
+    {
+      name: "state",
+      description: "(optional) state or territory",
+      type: "string",
+      group: "location",
+    },
+    {
+      name: "geolocation",
+      type: "geopoint",
+      group: "location",
+      description: "WGS84 Lat and Lng coordinates (decimal degrees).",
+    },
+    defineField({
+      type: "file",
+      name: "boundary",
+      group: "location",
+      description:
+        "(optional) GeoJSON Polygon or MultiPolygon representing the project boundary. .json files only.",
+      options: {
+        accept: "application/json",
+      },
+      validation: (r) =>
+        r.custom((a, b) => {
+          console.log(a, b);
```

Comment:
this is a mistake probably copypasta. Remove this validation field for `boundary`